src/NuGet.Core/NuGet.Configuration/PackageSourceMapping/PackageSourceMapping.cs

```diff
@@ -18,7 +18,7 @@ public class PackageSourceMapping
         /// <summary>
         /// Source name to package patterns list.
         /// </summary>
-        internal IReadOnlyDictionary<string, IReadOnlyList<string>> Patterns { get; }
+        public IReadOnlyDictionary<string, IReadOnlyList<string>> Patterns { get; }
```

Comment:
Do we need this change here yet?

Without reviewing the other PR it isn't immediately obvious that you'd be using this.

Reply:
Turns out I don't need it. Thanks for catching that